languages/elixir/exercises/concept/charlists/.docs/after.md

```diff
@@ -0,0 +1,162 @@
+Charlists are created using single quotes.
+
+```elixir
+'hello'
+```
+
+A charlist is a list of integers where the integers represent the Unicode values of a given character — also known as code points.
+
+## Charlists _are_ lists
+
+```elixir
+'' === []
+# => true
+
+is_list('hello')
+# => true
+```
+
+Because charlist are list, you can work with them just like with any other list - using recursion and pattern matching, or using the [`List`][list] module.
+
+```elixir
+[first_letter | _] = 'cat'
+first_letter
+# => 99
+
+List.first('hello')
+# => 104
+List.pop_at('hello', 0)
+# => {104, 'ello'}
+
+```
+
+You can concatenate two lists together using [`++`][kernel-concat-list].
+
+```elixir
+'hi' ++ '!'
+# => 'hi!'
+```
+
+The longer the first list is, the slower the concatenation, so avoid repeatedly appending to lists of arbitrary length.
+
+## Printability
+
+If a list of integers contains only integers that are code points of printable character, it will be displayed as a charlist. Even if it was defined using the `[]` syntax.
+
+```elixir
+'ABC'
+# => 'ABC'
+
+[65, 66, 67]
+# => 'ABC'
+```
+
+If a list of integers contains even one code point of an unprintable character (e.g. `0-6`, `14-26`, `28-31), it will be displayed as a list. Even if it was defined using the`''` syntax.
```

Comment:
```suggestion
If a list of integers contains even one code point of an unprintable character (e.g. `0-6`, `14-26`, `28-31`), it will be displayed as a list. Even if it was defined using the`''` syntax.
```